libr/search/search.c

```diff
@@ -9,6 +9,12 @@
 
 R_LIB_VERSION (r_search);
 
+typedef struct {
+	ut64 end;
+	int len;
+	ut8 data[];
```

Comment:
I'd like to accept a feature if compiles on gcc, clang and MSVC 2015.
Without  flexible array member, you'll do two `malloc`, the second for `ut8 *data`, and if it returned `NULL`, you had to clean the struct and return `NULL`.